### ISSUE TITLE
Clarification on Input Dimensions and Feature-based Iteration in model

### ISSUE THREAD

**Issue by S1mple0107** [swjtuer0762/xLSTMTime#4]
Thanks for your project and time!!
i find that the input in the project is (batch_size, features, context_points). for example, etth1 dataset's input is (batch, features=7, context_points=512).
sLSTMBlock's input is (batch, features=7, embedding_dim=256), which cause to the steps in slstm_forward.cu is featrues=7. According to the principle of RNN-type models, this iteration count should be related to the time steps.
Through my experiments, I found that your method can achieve good results. I don't know why the loop operation based on variables can yield good outcomes. I think the input should be (batch, context_points=512, features=7). By using slstm, the latest output h_t hidden state is obtained, and then the target output (batch, target_points, features) is obtained through multi-task learning.
could you give me some advice???

**Comment by S1mple0107** [comment]
I noticed that the loss value of the validation set reached its optimum in the first few epochs and then rose monotonically thereafter. Considering this is an indication of overfitting, do you have a similar situation?
